### PR TITLE
Related products

### PR DESCRIPTION
**Pull request by keihsu**
added some tests

### CODE REVIEW

File: spec/Related.spec.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+describe('Outfit Card', () => {
+  configure({ adapter: new Adapter() });
+  let wrapper;
+  beforeEach(() => {
+    wrapper = mount(
+      <Provider store={store}>
+        <ProductCard productId={16056} cardType="outfit"/>
+      </Provider>);
+    wrapper.update();
+    console.log(wrapper.debug());
```
Comment (ChristianStutzman) [inline spec/Related.spec.js]: Can you take out this console.log?

File: spec/Related.spec.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+describe('Related Product Card', () => {
+  configure({ adapter: new Adapter() });
+  let wrapper;
+  beforeEach(() => {
+    wrapper = mount(
+      <Provider store={store}>
+        <ProductCard productId={16056} cardType="related"/>
+      </Provider>);
+    wrapper.update();
+    console.log(wrapper.debug());
```
Comment (ChristianStutzman) [inline spec/Related.spec.js:45]: Can you take out this console.log for the merge?

**Review by ChristianStutzman — CHANGES_REQUESTED** [review]
Looks good! Can you take out those console.logs and I'll merge it? Also, I'm getting an error for multiple of the same keys in the Comparison component, looks to be from "charList" on line 96. Minor issue but can you look into that too?

**Comment by keihsu** [comment]
changes made!

**Review by ChristianStutzman — APPROVED** [review]

**Pull request merged**